contents/docs/privacy/data-collection.mdx

```diff
@@ -0,0 +1,225 @@
+---
+title: Controlling data collection
+sidebarTitle: Data collection
+sidebar: Docs
+showTitle: true
+---
+
+PostHog offers a range of controls to help you manage data collection. This guide covers data collection controls available to you **before** data reaches PostHog servers.
+
+You should consider the following tools to help you manage data collection:
+
+| Feature | Description |
+|---------|-------------|
+| [Asking for opt out](#asking-for-opt-out) | A top level opt out of all data collection |
+| [Autocapture behavior](#autocapture) | Configure what elements and interactions are automatically captured |
+| [Masking sensitive information](#masking-sensitive-information) | Prevent specific sensitive data from being collected |
+| [Overriding captured events](#overriding-captured-events) | Modify or filter event data before it's sent to PostHog |
+| [Cookieless tracking](#cookieless-tracking) | Track users without using browser cookies |
+| [Identifying users](#identifying-users) | Set up user identification and manage user properties |
+
+If you require that certain data never reaches PostHog servers, you can use one of the tools below to block data from being captured.
```

Comment:
```suggestion
If you require that certain data never reaches PostHog servers, you can use one of the tools below to prevent data from being captured.
```